### PR TITLE
Animations and Models

### PR DESCRIPTION
**Pull request by OroArmor**

### CODE REVIEW

File: mappings/net/minecraft/client/render/animation/Animatior.mapping [hunk 1]
```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/unmapped/C_qrwoclyv net/minecraft/client/render/animation/Animatior
```
Comment (triphora) [inline mappings/net/minecraft/client/render/animation/Animatior.mapping]:
```suggestion
CLASS net/minecraft/unmapped/C_qrwoclyv net/minecraft/client/render/animation/Animator
```

Comment (triphora) [reply mappings/net/minecraft/client/render/animation/Animatior.mapping]: remember to change file name too

File: mappings/net/minecraft/client/render/entity/model/FrogEntityModel.mapping [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	FIELD f_pckzsnyu tongue Lnet/minecraft/unmapped/C_rglqxnbw;
+	FIELD f_pnsrphfg rightLeg Lnet/minecraft/unmapped/C_rglqxnbw;
+	METHOD <init> (Lnet/minecraft/unmapped/C_rglqxnbw;)V
+		ARG 1 model
+	METHOD m_vbxzsuly animate (Lnet/minecraft/unmapped/C_kxntavoz;Lnet/minecraft/unmapped/C_mnzbondw;J)V
+		ARG 1 timer
+		ARG 2 animation
+		ARG 3 currentMillis
+	METHOD m_ztjmwfuh (Lnet/minecraft/unmapped/C_mnzbondw;JLnet/minecraft/unmapped/C_kxntavoz;)V
+		ARG 4 _timer
```
Comment (triphora) [inline mappings/net/minecraft/client/render/entity/model/FrogEntityModel.mapping:19]: Why does this have an underscore?

**Review by triphora — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/animation/Animation.mapping [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	METHOD m_qreyasec looping ()Z
+	CLASS C_mopdmkvg Builder
+		FIELD f_gtssoloz length F
+		FIELD f_mvezlfpr looping Z
+		FIELD f_roidfoyi animations Ljava/util/Map;
+		METHOD <init> (F)V
+			ARG 1 length
+		METHOD m_bwzckrkv build ()Lnet/minecraft/unmapped/C_mnzbondw;
+		METHOD m_loqmbnrq (Ljava/lang/String;)Ljava/util/List;
+			ARG 0 _part
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/animation/Animation.mapping:22]: underscore?
```suggestion
			ARG 0 part
```

File: mappings/net/minecraft/client/render/animation/Animatior.mapping [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+	METHOD m_dcgpwvbl scale (DDD)Lnet/minecraft/unmapped/C_zviwfwgf;
+		ARG 0 x
+		ARG 2 y
+		ARG 4 z
+	METHOD m_fyhlfhpd getAnimationTimestamp (Lnet/minecraft/unmapped/C_mnzbondw;J)F
+		ARG 0 animation
+		ARG 1 millis
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/animation/Animatior.mapping]: Is this milliseconds?
If so, it seems like an odd choice, I'd expect either the full word or `ms`.

File: mappings/net/minecraft/client/render/entity/model/FrogEntityModel.mapping [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	FIELD f_mwqcekod body Lnet/minecraft/unmapped/C_rglqxnbw;
+	FIELD f_nsewipyk eyes Lnet/minecraft/unmapped/C_rglqxnbw;
+	FIELD f_pckzsnyu tongue Lnet/minecraft/unmapped/C_rglqxnbw;
+	FIELD f_pnsrphfg rightLeg Lnet/minecraft/unmapped/C_rglqxnbw;
+	METHOD <init> (Lnet/minecraft/unmapped/C_rglqxnbw;)V
+		ARG 1 model
+	METHOD m_vbxzsuly animate (Lnet/minecraft/unmapped/C_kxntavoz;Lnet/minecraft/unmapped/C_mnzbondw;J)V
+		ARG 1 timer
+		ARG 2 animation
+		ARG 3 currentMillis
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/render/entity/model/FrogEntityModel.mapping:17]: same milliseconds question as my other comment

File: mappings/net/minecraft/client/render/entity/model/FrogEntityModel.mapping (hunk 2 above)
Comment (OroArmor) [reply mappings/net/minecraft/client/render/entity/model/FrogEntityModel.mapping:19]: It's a lambda variable that is the same as another value in the lvt. The `_` just indicates that it's local to the lambda only

File: mappings/net/minecraft/client/render/animation/Animation.mapping (hunk 3 above)
Comment (OroArmor) [reply mappings/net/minecraft/client/render/animation/Animation.mapping:22]: See my response to wafflecoffee

**Review by NoComment1105 — COMMENTED** [review]
Same as what others have said, otherwise good

File: mappings/net/minecraft/client/render/animation/Animatior.mapping (hunk 4 above)
Comment (Kroppeb) [reply mappings/net/minecraft/client/render/animation/Animatior.mapping]: `millis` is a common abbreviation.

Comment (supersaiyansubtlety) [reply mappings/net/minecraft/client/render/animation/Animatior.mapping]: ok

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/client/render/animation/Animatior.mapping (hunk 4 above)
Comment (OroArmor) [reply mappings/net/minecraft/client/render/animation/Animatior.mapping]: clarified to `elapsedMillis`

File: mappings/net/minecraft/client/render/entity/model/FrogEntityModel.mapping (hunk 5 above)
Comment (OroArmor) [reply mappings/net/minecraft/client/render/entity/model/FrogEntityModel.mapping:17]: follows `System.currentTimeMillis()`

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by triphora — APPROVED** [review]

**Review by IotaBread — APPROVED** [review]

**Pull request merged**